queue_services/entity-pay/devops/vaults.gcp.env

```diff
@@ -1,12 +1,14 @@
+AUDIENCE="op://gcp-queue/$APP_ENV/payment/AUDIENCE"
+ENTITY_FILER_TOPIC="op://gcp-queue/$APP_ENV/payment/ENTITY_FILER_TOPIC"
+ENTITY_MAILER_TOPIC="op://gcp-queue/$APP_ENV/payment/ENTITY_MAILER_TOPIC"
```

Comment:
I think this should be the following?  i.e. using the "topics" section

```
ENTITY_FILER_TOPIC="op://gcp-queue/$APP_ENV/topics/ENTITY_FILER_TOPIC"
ENTITY_MAILER_TOPIC="op://gcp-queue/$APP_ENV/topics/ENTITY_MAILER_TOPIC"
```

Reply:
I see, so there's a vault for the topics. I thought each one would be in it's respective project. OK, I'll fix that too for the other vaults.gcp.env.